---
uint8 status
string msg
